plots/measure.js

```diff
@@ -119,46 +167,112 @@ const URLS = [
   'https://bbc.com'
 ];
 
-/**
- * Launches Chrome once at the beginning, runs all the analysis,
- * and then kills Chrome.
- * TODO(chenwilliam): measure the overhead of starting chrome, if it's minimal
- * then open a fresh Chrome instance for each run.
- */
+function getUrls() {
+  if (SITES_PATH) {
+    return require(path.resolve(__dirname, SITES_PATH));
+  }
+
+  if (SITE) {
+    return [SITE];
+  }
+
+  if (SUBSET) {
+    return [
+      'https://en.wikipedia.org/wiki/Google',
+      'https://mobile.twitter.com/ChromeDevTools',
+      'https://www.instagram.com/stephencurry30',
+      'https://amazon.com',
+      'https://nytimes.com',
+      'https://www.google.com/search?q=flowers',
+
+      'https://flipkart.com',
+      'http://www.espn.com/',
+      'https://www.washingtonpost.com/pwa/',
+      'http://www.npr.org/',
+      'http://www.booking.com/',
+      'https://youtube.com',
+      'https://reddit.com',
+      'https://ebay.com',
+      'https://stackoverflow.com',
+      'https://apple.com',
+
+      // Could not run nasa on gin3g
+      'https://www.nasa.gov/',
+    ];
+  }
+
+  return SITES;
+}
+
+const URLS = getUrls();
+
 function main() {
   if (utils.isDir(constants.OUT_PATH)) {
     console.log('ERROR: Found output from previous run at: ', constants.OUT_PATH);
     console.log('Please run: npm run clean');
     return;
   }
 
-  const launcher = ChromeLauncher.launch();
-
-  return launcher.then(() => runAnalysis())
-  .catch(err => console.error(err))
-  .then(() => launcher.kill());
+  if (REUSE_CHROME) {
+    ChromeLauncher.launch({port: 9222}).then(launcher => {
```

Comment:
I've been running multiple plots at a time recently.. Launcher is getting a unique port by default, but you have to get it off `launcher.port` and pass into `lighthouse()`.. Here's what I did today: 
![image](https://cloud.githubusercontent.com/assets/39191/26388317/7535e0ca-4008-11e7-93a9-4d99330a4cfc.png)


Can you support this functionality?  You could just pass it along like this or create measure instances that hold this state.. wdyt?